### PR TITLE
infra(sentry): remove sentry integration

### PR DESCRIPTION
**Pull request by brynjarorng**
# Remove Sentry support

https://github.com/island-is/island.is/issues/8867

## What

Removing Sentry support

## Why

This will  reduce operational costs

## Checklist:

- [ ] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Formatting passes locally with my changes
- [ ] I have rebased against main before asking for a review

### CODE REVIEW

**Review by eirikurn — APPROVED** [review]
Reviewed core files.

**Review by HelgiHH — APPROVED** [review]
financial-statement files lgtm

**Review by disaerna — APPROVED** [review]

**Review by oddsson — APPROVED** [review]

**Review by RunarVestmann — APPROVED** [review]
Web files look good to me

File: libs/application/ui-shell/src/lib/FormShell.tsx [hunk 1]
```diff
@@ -105,57 +104,7 @@ export const FormShell: FC<{
...
-                    scope.setExtra('currentScreen', currentScreen.id)
-                  }}
-                  fallback={
-                    <ErrorShell
-                      title={formatMessage(coreMessages.globalErrorTitle)}
-                      subTitle={formatMessage(coreMessages.globalErrorMessage)}
-                    />
-                  }
-                >
-                  <Screen
```
Comment (obmagnusson) [inline libs/application/ui-shell/src/lib/FormShell.tsx:123]: We are going to need this screen element to display the application form. We just need to remove `<Sentry.ErrorBoundary>` that is wrapped around it.

**Review by obmagnusson — CHANGES_REQUESTED** [review]

**Review by davidpxn — APPROVED** [review]

**Review by pshomov — APPROVED** [review]
LGTM(DevOps)

File: libs/application/ui-shell/src/lib/FormShell.tsx (hunk 1 above)
Comment (brynjarorng) [reply libs/application/ui-shell/src/lib/FormShell.tsx:123]: Re-added wrapped code

**Review by albinagu — APPROVED** [review]
júní files 👍

**Review by obmagnusson — APPROVED** [review]
Application system files are LGTM

**Review by essgje — APPROVED** [review]
lftm

**Pull request merged**